src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -2890,6 +2895,17 @@ theorem getLsbD_intMax (w : Nat) : (intMax w).getLsbD i = decide (i + 1 < w) :=
   · simp [h]
   · rw [Nat.sub_add_cancel (Nat.two_pow_pos (w - 1)), Nat.two_pow_pred_mod_two_pow (by omega)]
 
+-- imp: two_pow_pred_lt_two_pow
+
+@[simp]
+theorem not_intMin {w : Nat} : intMax w = ~~~intMin w := by
+  ext i
```

Comment:
Do we really want to rewrite ~~~intMin to IntMax and not the other way around?

Reply:
the theorem is also never used, I can remove it from the current PR if we want to think about it more carefullt

Reply:
Yes, maybe that's a good idea. This PR is already complex, so let's try to minimize unrelated changes.